roles/mirror_ocp_release/tasks/registry.yml

```diff
@@ -25,7 +25,7 @@
 
 - name: Generate Image Source manifest
   ansible.builtin.command: >-
-    {{ mor_cache_dir }}/{{ mor_version }}/oc adm release mirror
+    {{ mor_oc }} adm release mirror
```

Comment:
Let's get back the validation of 4.14 here if we are mirroring a release lower than 4.14 we can try with `--dry-run=true` worst case we re-run it again

```yaml
    {{ mor_version is version("4.14", "<") | ternary("", "--print-mirror-instructions=" + mor_is_type | lower ) }}
```